cmd/collector/app/server/grpc_test.go

```diff
@@ -35,6 +36,13 @@ import (
 	"github.com/jaegertracing/jaeger/proto-gen/api_v2"
 )
 
+func stopOpenCensus(t *testing.T) {
```

Comment:
this is wrong. We don't call opencensus so should not be stopping it manually either. It's most likely some other component starts it and we're not shutting down that component.

Reply:
So, I removed the code.

The reason why I added it is that [opencensus has an init](https://github.com/census-instrumentation/opencensus-go/blob/01e6da5fc01c42aca1e0ce315f41744876e9fcbb/stats/view/worker.go#L33) that starts the worker 


I have searched for [NewMeter](https://github.com/census-instrumentation/opencensus-go/blob/01e6da5fc01c42aca1e0ce315f41744876e9fcbb/stats/view/worker.go#L261) within this codebase, the best that I could find is inside of the cmd/collector/app/handler using [opentelemtry's NewMeterProvider](https://pkg.go.dev/go.opentelemetry.io/otel/metric@v1.21.0/noop#NewMeterProvider) but none of these exposes the worker.

Here is a screenshot of the error:

<img width="845" alt="image" src="https://github.com/jaegertracing/jaeger/assets/30846935/a3c2a797-cf0b-4840-b849-b546c26be1ce">

Any suggestion here, I sincerely don't want to silent the leak.

Reply:
I prefer to silence it. Starting goroutines in init() is a stupid pattern and it does not indicate a leak in our code, so why bother handling it manually. See #5066 for similar issue.

I believe newer versions of OTEL Collector will soon remove opencensus dependency altogether.

Reply:
Fair enough